docker-compose.yaml

```diff
@@ -0,0 +1,49 @@
+version: '3'
+
+services:
+    php:
+        depends_on:
+          - elasticsearch1
+          - pgsql
+          - mariadb-server
+        image: rollerworks/search-dev:latest
+        volumes:
+            - .:/srv/www:rw
+
+    # Storage services
+    pgsql:
+        image: postgres:9.6-alpine
+        environment:
+            - POSTGRES_DB=search_tests
+            - POSTGRES_USER=postgres
+        volumes:
+            - pg-data:/var/lib/postgresql/data:rw
+        ports:
+            - "5432:5433"
```

Comment:
Is this an error? Exporting a port on a off-by-one port?

Reply:
This is only used for the host. Some people (including me) have a local instance running.
The docker instances still use the original port.

In fact I have been pulling out my hairs why the hell I can't connect to this server instance 💀 or MariaDB for that matter.

Reply:
Yeah, I thought so. If you're only changing the port to avoid collisions, I'd go with something **way** off instead of just off by one (because it might be considered a typo).

For example, prefixing all ports with 5?

MySQL: 53306
PgSQL: 55432
Elastic: 59200
etc.

That way you see from a mile away it's very different, there's a really low chance of collisions and you can actually document the difference easily. :)